### PR TITLE
Let any item smash windows instead of just weapons.

### PR DESCRIPTION
**Pull request by Anasari**
As per title.

🆑Ansari
tweak: All items can be used to smash windows instead of only those defined as weapons. (e.g. including guitar & flashlight)
/🆑

### CODE REVIEW

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -179,7 +179,7 @@ var/global/wcCommon = pick(list("#379963", "#0d8395", "#58b5c3", "#49e46e", "#8f
 	attack_generic(user, rand(10, 15))
 
 
-/obj/structure/window/attackby(obj/item/weapon/W as obj, mob/living/user as mob, params)
+/obj/structure/window/attackby(obj/item/W as obj, mob/living/user as mob, params)
```
Comment (Fox-McCloud) [inline code/game/objects/structures/window.dm]: I'm going to be a stickler and say this need to be `obj/item/I`.

I know that entails a lot more changes, but this kinda goes against the standard already established.

**Comment by Anasari** [comment]
Changed to comply with standard

**Comment by coldud13** [comment]
so prisoners will be able to break out of their cells with their shoes now?

**Comment by DesolateG** [comment]
Break out of prison with my ID

**Comment by Anasari** [comment]
They have no force. If it can be used to bash stuff, then it should be able to bash a window, if it cannot be used to bash stuff, then it shouldn't have any force.

if somehow ID / shoes can break windows then that's their problem

**Comment by MarcellusPye** [comment]
If they're an issue ziptie bucklecuff em, works for me.

**Comment by Ty-Omaha** [comment]
I can smash windows with my seclite.
10/10

**Comment by taukausanake** [comment]
Do gloves have force or just the slap emote? I'd love to challenge the window until it breaks apart

**Pull request merged**